frontend/src/components/HeaderBar.module.css

```diff
@@ -0,0 +1,19 @@
+.headerBar {
+  position: sticky;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: auto;
```

Comment:
does height auto need to be specified or is this for legibility purposes?

Reply:
I believe `auto` is the default value ([MDN docs](https://developer.mozilla.org/en-US/docs/Web/CSS/height) confirm this) but yeah I like to be as explicit as possible for setting width and height. It just saves other developers a moment of wondering if you meant to omit one or just forgot it